<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerNCIRayCastVolumeRenderingPropertiesWidget</class>
 <widget class="QWidget" name="qSlicerNCIRayCastVolumeRenderingPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NCI RayCast</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="DistColorBlendingLabel">
     <property name="text">
      <string>Dist. Color Blending:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ctkSliderWidget" name="DistColorBlendingSliderWidget">
     <property name="toolTip">
      <string>Distance Color Blending: Voxels with longer distance to eye/camera would be more darker to reveal depth information in volume. Higher value indicates stronger darking effect. Setting the value to 0 will turn off the effect.</string>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="pageStep">
      <double>0.200000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ICPEScaleLabel">
     <property name="text">
      <string>ICPE Scale:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ctkSliderWidget" name="ICPEScaleSliderWidget">
     <property name="toolTip">
      <string>Parameter scale (kt) for Illustrative Context Preserving Exploration, which simulates depth penetrating into volume.</string>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="pageStep">
      <double>4.000000000000000</double>
     </property>
     <property name="maximum">
      <double>20.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="ICPESmoothnessLabel">
     <property name="text">
      <string>ICPE: Smoothness:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ctkSliderWidget" name="ICPESmoothnessSliderWidget">
     <property name="toolTip">
      <string>Parameter smoothness (ks) for Illustrative Context Preserving Exploration.</string>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="pageStep">
      <double>0.200000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="DepthPeelingLabel">
     <property name="text">
      <string>Vol. Depth Peeling:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="ctkSliderWidget" name="DepthPeelingSliderWidget">
     <property name="toolTip">
      <string>Set threshold for Volumetric Depth Peeling. Volume rendering starts after we have met scalar values higher than the threshold. Use with transfer functions together.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="RenderingTechniqueLabel">
     <property name="text">
      <string>Technique (bg):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="RenderingTechniqueComboBox">
     <property name="toolTip">
      <string>Select Ray casting technique for the background (default) volume.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
